app/upgrades/v13/constants.go

```diff
@@ -5,7 +5,7 @@ package v13
 
 const (
 	// UpgradeName is the shared upgrade plan name for mainnet
-	UpgradeName = "v13.0.0-rc1"
+	UpgradeName = "v13.0.2"
```

Comment:
I know we usually don't change this but on `main` there is the `-rc1` suffix included in the constants currently - I would propose to rather update it to v13.0.2 or to the standard v13.0.0. WDYT?

Reply:
Isn't is suppose to be the latest e.g. `v13.0.2` ?

Reply:
we didn't backport these changes to `main` in the past but I would agree that it makes sense